projects/VersaGames/config.js

```diff
@@ -0,0 +1,146 @@
+const config = {
+  treasuryAddress: "0x5505ad96d4B1181c00248C8ac2a9A865065FD69D",
+  opsAddress: "0x3dABd1Ac5E89e01e928fca232672c4848C6E3097",
```

Comment:
looks like this is also EOA, sorry we cant count this towards tvl or treasury

Reply:
removed it